### DISCUSSION TITLE
Azure OpenAI

### DISCUSSION THREAD

**Discussion by mossyhub** [blakeblackshear/frigate#14083] [Ask A Question] [answered]
I see (with MASSIVE excitement) that you are working on GenAI for snapshot processing. Can you please also expose the AzureOpenAI within the OpenAI python library for people like me who have subscriptions there? https://github.com/openai/openai-python?tab=readme-ov-file#microsoft-azure-openai

**Comment by hawkeye217** [comment]
None of the other maintainers or myself have a subscription there, so unfortunately we have no way of testing any code we could write.

**Reply by mossyhub** [reply]
i would be happy to give you an API to mine for testing.

**Reply by hawkeye217** [reply]
Can you send me a DM on Discord? My username is the same as here.

**Reply by mossyhub** [reply]
done. lmoss26

**Comment by hawkeye217** [comment, marked as answer]
Implemented in https://github.com/blakeblackshear/frigate/pull/14102
